Snacks/Settings/SnacksProperties.cs

```diff
@@ -239,10 +239,11 @@ public static int NapTime
                         return 120;
 
                     case FaintTime.OneDay:
-                        if (GameSettings.KERBIN_TIME)
-                            return 360;
-                        else
-                            return 1440;
+                        //if (GameSettings.KERBIN_TIME)
+                        //    return 360;
+                        //else
+                        //    return 1440;
+                        return KSPUtil.dateTimeFormatter.Day;
```

Comment:
```suggestion
                        return KSPUtil.dateTimeFormatter.Day / 10;
```

The previous numbers look to be a tenth of a Day, could use a var here

Reply:
This may need author input; the series of settings are in order of increasing duration in minutes, with the change intended to fill in for a whole day. I'm not sure what you mean about a tenth of a day?

Reply:
Sorry mate I got my maths wrong, its not a fraction of a day like I thought - too many thoughts, not enough brain cells